src/dombeck_lab_to_nwb/azcorra2023/interfaces/azcorra2023_fiberphotometryinterface.py

```diff
@@ -0,0 +1,216 @@
+from pathlib import Path
+from typing import Optional
+
+from hdmf.backends.hdf5 import H5DataIO
+from hdmf.common import DynamicTableRegion
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.tools import get_module
+from neuroconv.utils import FilePathType
+from pymatreader import read_mat
+from pynwb import NWBFile
+
+
+class Azcorra2023FiberPhotometryInterface(BaseDataInterface):
+    """Data interface for Azcorra2023 fiber photometry data conversion."""
+
+    display_name = "Azcorra2023BinnedPhotometry"
+    associated_suffixes = ".mat"
+    info = "Interface for fiber photometry data."  # TBD
+
+    def __init__(
+        self,
+        file_path: FilePathType,
+        session_id: str,
+        fiber_depth_red: Optional[float] = None,
```

Comment:
Note that if there are two fibers, the depth of the second fiber is not in the binned photometry file. I'm going to change this logic in https://github.com/catalystneuro/dombeck-lab-to-nwb/pull/10/commits/4a590011d12e7358b692dc879c349e2a159e0e90, to set it as a `conversion_option` instead init parameter.